cleanlab/datalab/issue_manager/duplicate.py

```diff
@@ -0,0 +1,244 @@
+# Copyright (C) 2017-2023  Cleanlab Inc.
+# This file is part of cleanlab.
+#
+# cleanlab is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cleanlab is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with cleanlab.  If not, see <https://www.gnu.org/licenses/>.
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, ClassVar, Dict, List, Optional, Union
+import warnings
+
+import numpy as np
+import numpy.typing as npt
+import pandas as pd
+from scipy.sparse import csr_matrix
+from sklearn.neighbors import NearestNeighbors
+from sklearn.utils.validation import check_is_fitted
+
+from cleanlab.datalab.issue_manager import IssueManager
+from cleanlab.datalab.knn import KNN
+
+if TYPE_CHECKING:  # pragma: no cover
+    from cleanlab.datalab.datalab import Datalab
+
+
+class NearDuplicateIssueManager(IssueManager):
+    """Manages issues related to near-duplicate examples."""
+
+    description: ClassVar[
+        str
+    ] = """A near-duplicate issue refers to two or more examples in
+        a dataset that are extremely similar to each other, relative
+        to the rest of the dataset.
+
+        This may be reflected in the high similarity of the examples'
+        feature embeddings if the examples are represented as vectors.
```

Comment:
```suggestion
    ] = """A (near) duplicate issue refers to two or more examples in
        a dataset that are extremely similar to each other, relative
        to the rest of the dataset.  The examples flagged with this issue
        may be exactly duplicated, or lie atypically close together when
        represented as vectors (i.e. feature embeddings).
```

Reply:
(you can adjust widths as needed). Let's keep all these descriptions as one small block of text, no extra vertical spacing in them as then they take up too much of the report and distract from the numbers of interest.